README.md

```diff
@@ -1,18 +1,24 @@
 # Journey_Lua_Modder
-A python GUI application aiming to make Lua injection to the game "Journey" easier.
+A python GUI application aiming to make Lua injection to the game "Journey" easier. 
 
-How to install:
+Also works in the game "Flower" with minor changes. (do not copy Tick.lua from here, instead just put the function details in the existing file)
+
+### How to install:
 
 Download JourneyLuaModder.py, TickHook.lua, and StartLua.lua to Journey root folder.
 
-Embed Tick.lua.gz into Journey.exe according to this tutorial made by Alazar88:
+Modify \_\_Tick.lua.gz with our changes and embed it into Journey.exe according to this tutorial made by Alazar88:
 
 https://thatgamecompanyfan.boards.net/thread/541/pc-lua-script-editing-injection
 
-(You need to get approved first to view modding forum)
+(You need to get approved in the modding forum to view the thread)
+
+__Optional recommendations:__
+- Patch your modding exe via steamless to disable Steam launching redirection, still requires Steam running. 
+- Use a emulator when modding to avoid impacting others gameplay.
 
-run JourneyLuaModder.py
+Requires Python 3, run pythonw JourneyLuaModder.py 
```

Comment:
After having troubles with working directories and python versions, I engage all users to stick to the release version which is in exe form.

Reply:
Please bundle the exe with latest python 3.8 because 3.9 drops support for windows 7